src/main/c/ssl.c

```diff
@@ -2036,6 +2036,17 @@ TCN_IMPLEMENT_CALL(void, SSL, setState)(TCN_STDARGS,
     SSL_set_state(ssl_, state);
 }
 
+TCN_IMPLEMENT_CALL(jint, SSL, release0)(TCN_STDARGS)
```

Comment:
is it guaranteed that SSLEAY_VERSION will return different things for libressl / openssl / boringssl ? Also where would the user find the values for each of the different releases ?